src/include/distributed/master_protocol.h

```diff
@@ -64,7 +64,12 @@
 	"SELECT worker_apply_inter_shard_ddl_command (" UINT64_FORMAT ", %s, " UINT64_FORMAT \
 	", %s, %s)"
 #define SHARD_RANGE_QUERY "SELECT min(%s), max(%s) FROM %s"
+#define INNER_SHARD_TABLE_SIZE_QUERY " pg_table_size(%s) + "
```

Comment:
I know I said let's add '+' here before. But I think it will be better if we just add '+' signs in the query generating function. You can also remove the spaces around pg_table_size and add them around '+'